nodes/mainnet.md

```diff
@@ -121,7 +124,7 @@ There are multiple explorers you can use for Mainnet Beta:
 - [https://celenium.io](https://celenium.io)
 - [https://celestiascan.com](https://celestiascan.com)
 - [https://mintscan.io/celestia](https://mintscan.io/celestia)
-- [https://celestia.explorers.guru](https://celestia.explorers.guru)
```

Comment:
this is out of scope for this PR. why are you removing this explorer?

Reply:
Oh, sorry, My mistake.
Fixed.